### DISCUSSION TITLE
deepin+Archlinux双系统下无法使用使用某些快捷键

### DISCUSSION THREAD

**Discussion by amit-gshe** [linuxdeepin/developer-center#6117] [Troubleshooting | 故障排查] [answered]
如题，archlinux新安装deepin桌面后，alt+tab不生效，控制中心配置了alt+tab切换窗口也不生效，
配置super+up切换窗口最大化也不生效，但如果把窗口最大化的快捷键改成alt+up则可以进行窗口最大化。

问题背景：我是安装的双系统deepin+archlinux，archlinux使用的是deepin系统的home目录，deepin系统还是20.9，archlinux是最新的v23，不确定是否是配置冲突。

请问我应该从哪些方面去排查问题出在哪里？

**Comment by amit-gshe** [comment]
先systemctl stop lightdm.service，然后把home目录下的.config和.local目录下deepin相关的文件全部重命名.bak后，再systemctl start lightdm.service，登录后快捷键就正常了。

**Reply by BLumia** [reply]
cc @justforlxz  看看是不是需要我们主动处理？

**Reply by justforlxz** [reply, marked as answer]
不好弄，实际上只需要删除一次 ~/.config/kglobalaccels 文件后重启kwin就可以了，但是无脑删除用户的配置文件这个行为不太好（（特别是kde+dde用户
